application/views/template_view.py

```diff
@@ -14,7 +14,12 @@ def upload_template(template_id):
 @template_view.route('/template/<template_id>', methods=['GET'])
 def get_template_by_id(template_id):
     template = template_controller.get_comms_template_by_id(template_id)
-
     http_code = 200 if template else 404
-
     return make_response(jsonify(template), http_code)
+
+
+@template_view.route('/template/<template_id>', methods=['DELETE'])
+def delete_template(template_id):
+    is_deleted = template_controller.delete_comms_template(template_id)
+    http_code = 200 if is_deleted else 204
```

Comment:
Is 204 appropriate here? It might be ok just not seen it used to determine whether something has been deleted or not it's normally used to inform the caller not to expect data in the response

Reply:
should be 404